recipe/meta.yaml

```diff
@@ -39,9 +40,12 @@ test:
 
   requires:
     - pytest >=3.0.5
-    - scikit-image >=0.12.3,<0.17.0
 
   commands:
+    # Disable newer Arrow strings with dask-image.
+    # xref: https://github.com/dask/dask-image/issues/335
+    - export DASK_DATAFRAME__CONVERT_STRING="False"  # [unix]
+    - set "DASK_DATAFRAME__CONVERT_STRING=False"     # [win]
     - pytest
```

Comment:
As noted above, using the newer Arrow-based strings added in Dask are causing test failures in dask-image. So just disable this option until we can figure out how to fix them

<hr>

The Dask config docs show [`dataframe.convert-string`]( https://docs.dask.org/en/stable/configuration.html#dataframe.convert-string ) as the option to use for configuring Arrow-based strings in Dask

Based on offline testing, this seems to be the right environment variable to set to affect the `dask.config`. Would note the default value is `None`, which IIUC means it will use Arrow-based strings if Arrow is available. So we can see that it is now `False`, which is what we need to disable them completely

```bash
$ DASK_DATAFRAME__CONVERT_STRING="False" python -c "import dask; print(dask.config.get('dataframe.convert-string'))"
False
```

Reply:
Appears to be working! 🎉